skbio/tree/tests/test_tree.py

```diff
@@ -784,6 +784,39 @@ def test_levelorder(self):
         obs = [n.name for n in self.simple_t.levelorder()]
         self.assertEqual(obs, exp)
 
+    def test_bifurcate(self):
+        t1 = TreeNode.read(StringIO(u'(((a,b),c),(d,e));'))
+        t2 = TreeNode.read(StringIO(u'((((((a,b),(c,d)),(e,f))J)Q)K);'))
+        t3 = TreeNode.read(StringIO(u'(((a,b,c),(d)),(e,f));'))
+        t4 = TreeNode.read(StringIO(u'(((a,b,c),(d)),(e,f));'))
+        t5 = TreeNode.read(StringIO(u'(((c,(f)d,(g,h)e)b)a);'))
+        t6 = TreeNode.read(StringIO(u'((((g,h)e,(f)d,c)b)a);'))
+        t7 = TreeNode.read(StringIO(u'((c,((g,h)e,f))b);\n'))
+        t8 = TreeNode.read(StringIO(u'((((g,h)e,c,(f)d)b)a);'))
+        t9 = TreeNode.read(StringIO(u'((((g,h)e,c,(f)d)b)a);'))
```

Comment:
`t8` and `t9` are identical, as are `t3` and `t4`. Could just use `.copy`

Possible to add a test case in which the number of tips in the tree is odd?